test/test_torch.py

```diff
@@ -4483,6 +4483,20 @@ def test_empty_meta(self):
             z = x + y
             self.assertEqual(z.size(), (2 ** 20, 2 ** 20))
 
+        def test_upsample_nearest1d_meta(self):
+            # TODO: this is not a sustainable way of testing meta functions,
+            # but I want some quick scaffolding first before a more
+            # integrated testing strategy
+            x = torch.empty_meta(2 * 10 ** 10, 3, 2 * 10 ** 10)
+            z = torch.nn.functional.interpolate(x, scale_factor=2)
+            self.assertEqual(z.size(), (2 * 10 ** 10, 3, 4 * 10 ** 10))
+
+            # interpolate doesn't seem to support out=
+            # (not sure why passing None here doesn't work? How strange...)
+            z = torch.empty_meta(0)
+            torch._C._nn.upsample_nearest1d(x, (4 * 10 ** 10,), 2, out=z)
+            self.assertEqual(z.size(), (2 * 10 ** 10, 3, 4 * 10 ** 10))
```

Comment:
maybe worth asserting that we didn't actually perform the computation? Also totally understand that the meta testing is subject to change.

Reply:
I'm not... sure exactly how to test that haha. I'm sort of trying to implicitly test this by using ludicrously large sizes here; if you actually allocated these you'd probably OOM.

Reply:
I was just thinking running the meta op, running the actual op, then asserting that their `out` results aren't equal (I'm not actually sure what the contents of the `empta_meta` are- if it's uninitialized memory we could just zero it out first).

Reply:
A meta tensor should give a predictable error if you try to access its data, right? Can we just test by expecting that error here?

In master `torch.empty_meta(10)[0]` gets you a RuntimeError:
```
RuntimeError: Could not run 'aten::as_strided' with arguments from the 'Meta' backend.
```

Expecting a RuntimeError on data access (not on `as_strided`, just in general) seems like a reasonable proxy for "didn't compute"